docs_src/security/tutorial004.py

```diff
@@ -98,6 +98,8 @@ async def get_current_user(token: str = Depends(oauth2_scheme)):
         if username is None:
             raise credentials_exception
         token_data = TokenData(username=username)
+    except ExpiredSignatureError:  # Check signature expiry
+        raise credentials_exception
```

Comment:
I would modify to:
```python
except ExpiredSignatureError as expired_signature:  # Check signature expiry
        raise credentials_exception from expired_signature
```
and/or follow @giladfuchs [suggestion](https://github.com/tiangolo/fastapi/pull/10406#issuecomment-1766126432) and raise a different exception in this case